TalkAloud/TalkAloud/Model/AudioEngine.swift

```diff
@@ -107,6 +108,7 @@ class AudioEngine: NSObject {
         audioState = .playing
     }
     
+    // TODO Change withFileURL to fetch url attribute
```

Comment:
What do you mean with this TODO?

Reply:
I forgot I made this but I was thinking at first that I was going to have to use the url property from CoreData. I thought I deleted this maybe this was from that commit I went back to.